File: my_code/DSA/PYTHON/Add1.py
import time

def add_up_to(n):
    result = 0
    for i in range(1, n+ 1):
        result += i
    return result

start_time = time.perf_counter()
print(add_up_to(1000000000))
end_time = time.perf_counter()
print(f'The running time is {(end_time - start_time)} seconds. ')

File: my_code/DSA/JS/Add1.js
function addUpTo(n) {
  let total = 0;
  for (let i = 0; i <= n; i++) {
    total += i;
  }
  return total;
}

let startTime = performance.now();
console.log(addUpTo(1000000000));
let endTime = performance.now();
console.log(`The running time is ${(endTime - startTime) / 1000} seconds. `);
